.github/workflows/gh-super-linter.yml

```diff
@@ -0,0 +1,18 @@
+name: Lint Code Base (GH SuperLinter)
+on: [push, pull_request]
+jobs:
+  build:
+    name: Lint whole code base
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Lint Code Base
+        uses: super-linter/super-linter@v4
+        env:
+          VALIDATE_DOCKERFILE_HADOLINT: true
```

Comment:
[https://github.com/marketplace/actions/super-linter#environment-variables](url)
It looks like out of the box every VALIDATE_[language] is set to true unless any of them are explicitly set, in which case everything else is set to false. So I think there's no point of explicitly setting em unless we have any languages we want to exclude down the line